Three/include/CGAL/Three/Scene_item.h

```diff
@@ -231,8 +231,21 @@ class SCENE_ITEM_EXPORT Scene_item : public QObject {
 
   //!Handles key press events.
   virtual bool keyPressEvent(QKeyEvent*){return false;}
+
+  //!Contains the header for the table in the statistics dialog
+  struct header_data{
+   //!Contains the name of the category of statistics and the number of lines it will contain
```

Comment:
Note: The naming convention in CGAL says that class must have the first letter capitalized. In Qt5 the coding 
convention is the the camel-case. That name `header_data` does not fix any convention.